low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -295,7 +295,7 @@ class CheckCmd(Cmd):
     def __init__(self, args):
         super().__init__(args)
 
-        from framework.base.sspl_constants import PRODUCT_FAMILY
+        from cortx.sspl.lowlevel.framework.base.sspl_constants import PRODUCT_FAMILY
```

Comment:
Please keep this import statement at top.

Reply:
As mentioned in other PR, import for these specific functions here are fine